README.md

```diff
@@ -20,7 +20,7 @@ so your PR will get merged faster, and you can start enjoying your shiny new ent
 We use the `domain`, `type` and `identifier` to assign each entity a Global Unique Identifier (GUID).
 - The `domain` must be a value matching `/[A-Z][A-Z0-9_]{2,7}/`. This field is mostly relevant internally for NR. 
 Use EXT by default, although we may advise to use a different value in some cases.                
-- The `type` must be a value matching `/[A-Z][A-Z0-9_]{2,11}/`. This field is meant to identify the type of entity. 
+- The `type` must be a value matching `/[A-Z][A-Z0-9_]{2,19}/`. This field is meant to identify the type of entity. 
```

Comment:
`KUBERNETES_APISERVER` was too long for this requirement, so I increased it. This will leave us less space for the identifier in the GUID, maybe that's why it was encoded @srvaroa?

Reply:
We should do the opposite actually, the sizes of each field are defined in the entity guid spec so we should instead make KUBERNETES_APISERVER shorter (my bad!)

Reply:
Btw we should copy (move?) the entity guid spec here.

Reply:
Oh, good, then I'll change that. I think that we discussed adding the whole spec but it had very internal information so we added the relevant parts of the spec to the docs, I can do another check and see if anything relevant is missing (or moving it to another file if it's clearer).

Reply:
Yeah you're right, looking in the text it really contains everything that's needed